config.json

```diff
@@ -176,6 +176,16 @@
       ]
     },
     {
+      "uuid": "8d345e34-e815-49cd-91ab-ca8db63b07ef",
+      "slug": "two-fer",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+
+      ]
+    },
+    {
```

Comment:
Please put new exercises at the end of the list.

Is this difficulty assessment accurate?

Please add some topics.